src/test/java/com/gavinfenton/quizolation/service/QuizServiceTest.java

```diff
@@ -144,6 +148,32 @@ public void testUpdateQuizSetsIdBeforeSaveAndReturnsFromRepository() {
         assertEquals(quizExpected, quizActual);
     }
 
+    @Test
+    public void testAddTeamToQuizCallsAndReturnsFromRepo() throws Exception {
```

Comment:
Just spotted, throws Exception isn't necessary as no exception will be thrown

Reply:
Done